ctags/parsers/c.c

```diff
@@ -290,12 +289,12 @@ static kindOption CKinds [] = {
 	{ true,  'g', "enum",       "enumeration names"},
 	{ true,  'm', "member",     "class, struct, and union members"},
 	{ true,  'n', "namespace",  "namespaces"},
-	{ false, 'p', "prototype",  "function prototypes"},
+	{ true,  'p', "prototype",  "function prototypes"},
```

Comment:
The thing is that the "enabled" field was completely ignored by ctags in Geany and everything was enabled.

Now thinking about it we should probably keep doing that - for a library it's more logical to  get all tags and let the code decide what to do with them. Also if we migrate to the cpp parser, we'd have to introduce diffs like that there too.